Doom-Mummies/Welcome-to-Nashville
# Rhc save results in itinerary

# Description

Completed my portion of the work with the help of my teammates. My search results are limited to displaying only four options. Of those, clicking save puts that one in the itinerary.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1) `git fetch --all`
2) `git checkout rhc-save-results-in-itinerary`
3) Run http-server
4) Type in a genre into the "concerts by genre" field and click the "Search" button
5) In the "Results" section, click a Save button to one of the options
6) Ensure the option is saved to the My Itinerary section

# Checklist:
